tools/install-source/Program.cs

```diff
@@ -43,27 +58,26 @@ public static void Main (string[] arguments)
 
 		var srcs = new HashSet<string> (StringComparer.OrdinalIgnoreCase);
 
-		if (!monopath.EndsWith (Path.DirectorySeparatorChar.ToString (), StringComparison.Ordinal))
-			monopath += Path.DirectorySeparatorChar;
+		monopath = FixPathEnding (monopath);
+		xamarinpath = FixPathEnding (xamarinpath);
+		opentkpath = FixPathEnding (opentkpath);
+
+		var manglerFactory = new PathManglerFactory {
+			InstallDir = installDir,
+			MonoSourcePath = monopath,
+			XamarinSourcePath = xamarinpath,
+			FrameworkPath = (installDir.Contains ("Xamarin.iOS.framework")) ? "Xamarin.iOS.framework" : "Xamarin.Mac.framework",
+			OpenTKSourcePath = opentkpath,
+		};
 
+		Console.WriteLine ($"Install dir is {installDir}");
 		foreach (string mdb_file in mdb_files) {
+			Console.WriteLine("Mdb file is {0}", mdb_file);
```

Comment:
You can even do the trick that mmp/mtouch does and make verbose a number and have some output that only shows up with multiple -v's